src/components/Thumbnail/ThumbnailArticle.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC } from "react";
+import { Button } from "./Button";
+import { Description } from "./Description";
+import { Image } from "./Image";
+import { Name } from "./Name";
+import { Date } from "./Date";
+import { Thumbnail } from "./Thumbnail";
+
+interface IProps {
+  date?: string;
+  name: string;
+  description: string;
+  button: string;
+  icon: string;
+  alt: string;
+}
+
+export const ThumbnailArticle: FC<IProps> = ({
+  date,
+  name,
+  description,
+  button,
+  icon,
+  alt
+}) => (
+  <Thumbnail>
+    <Image icon={icon} alt={alt} className="mb-8" />
+    <Date>{date}</Date>
```

Comment:
if prop is optional, you need handle state if prop is optional, but this props shouldn't be optional